src/CommunityToolkit.Maui.Core/Essentials/SpeechToText/SharedSpeechToTextImplementation.macios.cs

```diff
@@ -21,6 +21,20 @@ public sealed partial class SpeechToTextImplementation
 												? SpeechToTextState.Listening
 												: SpeechToTextState.Stopped;
 
+	/// <summary>
+	/// Initialize <see cref="SpeechToTextImplementation"/>
+	/// </summary>
+	public SpeechToTextImplementation()
+	{
+		var audioSession = AVAudioSession.SharedInstance();
+		if (UIDevice.CurrentDevice.CheckSystemVersion(15, 0))
+		{
+			audioSession.SetSupportsMultichannelContent(true, out _);
+		}
+
+		audioSession.SetCategory(AVAudioSessionCategory.PlayAndRecord, AVAudioSessionCategoryOptions.DefaultToSpeaker);
```

Comment:
Should we set it once or each time before starting the recognition?

Reply:
Ah - yea, good point, we should be setting it every time speech recognition is triggered. I can update it in the morning.

Reply:
Thanks again, Vlad! Updated ✅